src/common/socket.h

```diff
@@ -9,7 +9,9 @@
 
 
 #ifdef WIN32
-    #define FD_SETSIZE 1000
+    #ifndef FD_SETSIZE
```

Comment:
this is just hiding the real problem, unfortunately (the real problem being that include order is not right - this should always be set before including winsock2, but something is including winsock2 before this that I haven't found yet)

this would just not override the fd size at all